### PR TITLE
Add transforms of node drags to drag methods

### PR DESCRIPTION
**Pull request by Alexithemia**
I am not sure if you have thought about or tried to implement multiple node dragging at one time, but for my project we have implemented the ability to select multiple nodes, and we would like to be able to move all the selected nodes at once while dragging.

So I made this PR to provide a transform object back for onNodeDrag and onNodeDrag end.

onNodeDrag provides every little transform change, while onNodeDragEnd provides a full distance transform from the original coordinates to the end coordinates.

I'm sure this may have other use applications, but I will be using this to edit the positions of all the nodes in my selected nodes array.

I am going to make a PR for the 2D version as well. Thanks!

### CODE REVIEW

File: src/3d-force-graph.js [hunk 1]
```diff
@@ -250,16 +251,17 @@ export default Kapsule({
             state.ignoreOneClick = true; // Don't click the node if it's being dragged
 
             const node = event.object.__data;
-
+            const newPos = event.object.position;
+            const transform = {x: newPos.x - node.x, y: newPos.y - node.y, z: newPos.z - node.z}
```
Comment (vasturiano) [inline src/3d-force-graph.js]: Missing `;`

File: src/3d-force-graph.js [hunk 2]
```diff
@@ -271,20 +273,21 @@ export default Kapsule({
               delete(node.__disposeControlsAfterDrag);
             }
 
-            const initPos = node.__initialFixedPos;
-
-            if (initPos) {
+            const initFixedPos = node.__initialFixedPos;
+            const initPos = node.__initialPos;
+            const transform = {x: initPos.x - node.x, y: initPos.y - node.y, z: initPos.z - node.z}
```
Comment (vasturiano) [inline src/3d-force-graph.js]: Missing `;` like above

File: src/3d-force-graph.js (hunk 1 above)
Comment (vasturiano) [inline src/3d-force-graph.js]: Also, "translate" may be a more suitable var name, instead of "transform".

**Comment by Alexithemia** [comment]
> Thanks for the PR Alex. It's a useful feature! Would you mind adding some docs to the `drag` events in the readme?

Sure thing

**Comment by vasturiano** [comment]
Thanks @Alexithemia!  🎉 🎉

**Pull request merged**